### DISCUSSION TITLE
Material para estudar RxJS

### DISCUSSION THREAD

**Discussion by synga** [frontendbr/forum#1412] [Discussão]
Fala galera.
To precisando/querendo aprender RxJS, mas to tendo uma dificuldade de achar bons conteúdos por ai, principalmente conteúdos mais fáceis de se utilizar e de entender (ouvi não sei onde, acho que no Hipsters.tec alguém falando que RxJS tem uma maldição que quando você aprende não consegue ensinar, deve ser isso hahahaha).

Por hora só conheço o [learnrxjs.io](https://www.learnrxjs.io/) que é legal, mas não tem uma ordem então não sei o que seria o básico a se aprender e quais operadores seriam mais avançados.

Trampo com Angular então se o material for relacionado ao Angular melhor, se não sem problemas. Se alguém puder ajudar então agradeço.

**Comment by waldemirflj** [comment]
@synga indico os links abaixo. Na época me foram util.

**Play**
https://www.youtube.com/watch?v=hrSYyr2sUac
https://www.youtube.com/watch?v=QGA0ApPtgWo
https://www.youtube.com/watch?v=2LCo926NFLI

**Playlist**
https://www.youtube.com/watch?v=T9wOu11uU6U&list=PL55RiY5tL51pHpagYcrN9ubNLVXF8rGVi

**Comment by rubenmarcus** [comment]
https://www.udemy.com/rxjs-reactive-angular-course/learn/
https://www.udemy.com/rxjs-course/

**Comment by vczb** [comment]
Oi, comecei a estudar nessa semana e achei esse post e to gostando bastante

https://www.bitovi.com/blog/rxjs-with-react-pt-1

**Comment by leandrocrs** [comment]
Passei pelo mesmo problema, e o que me salvou na época foi esse curso:

https://www.udemy.com/share/101YpgAkESc1pVQHw=/

Recomendo os outros sobre Angular do mesmo autor, o inglês dele é fácil de compreender pra não-nativos e ele é muito didático.

**Comment by vczb** [comment]
Um tec talk legal sobre RxJS com o Daciuk

https://www.youtube.com/watch?v=82JphanBa_A
